Server/shop/src/main/java/com/cashmanager/server/shop/OrderedOrder.java

```diff
@@ -0,0 +1,48 @@
+package com.cashmanager.server.shop;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.util.Objects;
+import java.util.UUID;
+
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@Builder
+@AllArgsConstructor
+@Entity
+@Table(name = "ordered_orders")
+public class OrderedOrder {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column( nullable = false)
+    private UUID orderedOrdersId;
```

Comment:
same remark for the primary key name
```suggestion
    private UUID id;
```